var/spack/repos/builtin/packages/rocfft/package.py

```diff
@@ -205,4 +211,7 @@ def cmake_args(self):
 
         if self.spec.satisfies("@5.3.0:"):
             args.append("-DCMAKE_INSTALL_LIBDIR=lib")
+        if self.spec.satisfies("@5.4.0:"):
+            args.append("-Dsqlite_INCLUDE_DIR={0}".format(self.spec["sqlite"].prefix.include))
+
```

Comment:
I've merged a fix to rocFFT upstream. IMO, it's a better fix because you can drop these four lines and it will still work. Not sure it will make it into ROCm 5.5, but it should be there for ROCm 6.0.

https://github.com/ROCmSoftwarePlatform/rocFFT/commit/f66855d5a337f3aa4e9be44fd639855fd301c9c0.patch